xplot.py

```diff
@@ -1 +1,49 @@
-#
+'''
+Problem 1: Plot Bar
+'''
+# TEST CASE GENERATOR # DO NOT MODIFY # 
+inputs_list = [["A", 627.2],["B", 38.9],["C", 11.3]]
+inputs_dict = {"A": 627.2, "B": 38.9,"C": 11.3}
+
+# SOLUTION
+
+def barGraph1(inputs_list):
+  max_val = max(inputs_list, key=lambda x: x[1])[1]
```

Comment:
can be replaced by `max(i[0] for i in inputs_list)`

Reply:
Ah, passing a generator through max. 
Thanks, this is definitely better.